snapcraft/internal/deltas/_xdelta.py

```diff
@@ -0,0 +1,48 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import logging
+import subprocess
+
+from ._deltas import BaseDeltasGenerator
```

Comment:
I'm not sure if I 100% understand the point, what I improved is to expose the needed the classes/function in the *snapcraft/internal/deltas/__init__.py*, then in the test class only * import snapcraft.internal*,  the use the class like: snapcraft.internal.deltas.XDeltaGenerator.

Correct me if I misunderstand you.